pkg/server/conn_stmt_params_test.go

```diff
@@ -35,17 +35,17 @@ import (
 	"github.com/stretchr/testify/require"
```

Comment:
Please add more tests at least including `COM_QUERY`, `COM_EXECUTE` cases with `query attributes` here. You can reference `TestDateTimeTypes` to write similar protocol tests.

It's painful to write pure binary protocol tests :facepalm:, but it can make our software much safer.

Reply:
Had add tests.

Reply:
In Wireshark I added "Copy as Go literal" support. This might make it easier to write some tests that depend on specific (parts of) network packets.

![image](https://github.com/user-attachments/assets/9f77727b-90c6-4cf5-aa80-983fcc3a2c04)